src/components/ui/header.jsx

```diff
@@ -11,10 +11,10 @@ const Header = () => {
       </Link>
       <nav className="ml-auto flex gap-4 sm:gap-6">
         <Link
-          href="#"
+          href="/Statistics"
           className="text-sm font-medium hover:text-[#000080] text-[#333333]"
           prefetch={false}>
-          Features
+          Statistics
         </Link>
```

Comment:
I could be wrong, but I think you want to remove this from the header. I THINK after the user logs in..there will be a separate page that Angela and Tal are working ..whatever header they use is where u would want the link to the statistics page to show up.